providers/efa/man/efadv_query_device.3.md

```diff
@@ -61,6 +63,13 @@ struct efadv_device_attr {
 *inline_buf_size*
 :	Maximum inline buffer size.
 
+*max_wr_rdma_sge*
```

Comment:
Isn't this a standard query already?

Reply:
According to the documentation:

>         int                     max_sge_rd;             /* Maximum number of s/g per WR for RD QPs */


This isn't quite an RD QP.

Reply:
max_sge is the max number of SGEs usable for all QPs types for everything but RDMA READ
max_sge_rd is the max number of SGEs usable for all QP types for RDMA READ

Where did you find that wrong comment?

Reply:
Ah, shouldn't have trust anything but the man in the repo..
https://linux.die.net/man/3/ibv_query_device

Thanks, I will remove this field.